integration-tests/api/medusa-config.js

```diff
@@ -15,6 +15,12 @@ module.exports = {
     cookie_secret: "test",
   },
   modules: {
+    eventBus: {
```

Comment:
Question: Have we identified why the local one created this issue? We might have an issue to fix somewhere otherwise the users might encounter the same issue no?

Reply:
nope, @olivermrbl will look into it after pre-release stuff I think

Reply:
https://github.com/medusajs/medusa/pull/3575